### DISCUSSION TITLE
test

### DISCUSSION THREAD

**Discussion by dwightskrute** [outline/outline#5882] [Self Hosting]
I'm installing Outline with the web documentation and I'm getting the error "econnrefused 127.0.0.1:5432". Outline docker does not start for me or takes several minutes.

The installation is to test it on a server without any domain, trying to access from localhost or some docker IP. Is this possible or is it mandatory to have the subdirectory?

I appreciate all the information you can give me for the docker installation, on a test server without having domains.

**Comment by tommoor** [comment]
It is mandatory to use a fully qualified domain.

**Reply by dwightskrute** [reply]
Is there no option to use it with localhost? or any way to test it?
